lib/archethic.ex

```diff
@@ -240,11 +240,34 @@ defmodule Archethic do
   """
   @spec get_transaction_chain(binary()) :: {:ok, list(Transaction.t())} | {:error, :network_issue}
   def get_transaction_chain(address) when is_binary(address) do
+    local_available_nodes = locally_available_nodes(address)
+    get_transaction_chain(local_available_nodes, address)
+  end
+
+  def get_transaction_chain_by_paging_address(address, paging_address) when is_binary(address) do
```

Comment:
Could you add typespec and doc as well ?

Reply:
Updated the function with doc and typespe